msresist/binomial.py

```diff
@@ -164,13 +163,13 @@ def from_summaries(self, inertia=0.0):
         betaA = np.sum(self.weightsIn) - k
         betaA = np.clip(betaA, 0.01, np.inf)
         probmat = sc.betainc(betaA, k + 1, 1 - self.background[0])
-        self.logWeights[:] = self.SeqWeight * np.log(np.tensordot(self.background[1], probmat, axes=2))
-        self.logWeights[:] = self.logWeights - np.mean(self.logWeights)
+        self.logWeights[:] = np.log(np.tensordot(self.background[1], probmat, axes=2))
+        self.logWeights[:] = self.logWeights - np.amax(self.logWeights)
```

Comment:
`amax` keeps the probabilities from being really small. This seems to help. I think there might be a conversion point in pom where a small LL is turned into 0.